src/main.js

```diff
@@ -30,13 +30,16 @@ requirejs.config({
             name: 'json',
             location: '../bower_components/requirejs-plugins/src',
             main: 'json.js'
-        }
+        },
+        {
+            name: "codemirror",
+            location: "../bower_components/codemirror",
+            main: "lib/codemirror",
+        },
     ],
     paths: {
         'vellum': '.',
 
-        'codemirror': '../lib/codemirror/xml',
-        'codemirrorBase': '../bower_components/codemirror/lib/codemirror',
```

Comment:
Oh, didn't see your comment before I posted my review. My guess is that you need to replace these removed lines with something similar, but pointing to the new version.